app/angular/src/builders/start-storybook/index.ts

```diff
@@ -40,7 +40,7 @@ export type StorybookBuilderOptions = JsonObject & {
 
 export type StorybookBuilderOutput = JsonObject & BuilderOutput & {};
 
-export default createBuilder(commandBuilder);
+export default createBuilder<any>(commandBuilder);
```

Comment:
maybe I can use a different type here. However, `StorybookBuilderOptions` was not working